plugins/net_plugin/net_plugin.cpp

```diff
@@ -4015,12 +4014,13 @@ namespace eosio {
       return fc::json::from_string(s).as<T>();
    }
 
-   size_t net_plugin_impl::parse_connection_limit( const std::string& limit_str) {
+   size_t net_plugin_impl::parse_connection_rate_limit( const std::string& limit_str) {
       std::istringstream in(limit_str);
-      fc_dlog( logger, "parsing connection endpoint with locale ${l}", ("l", std::locale("").name()));
+      fc_dlog( logger, "parsing connection endpoint limit ${limit} with locale ${l}", ("limit", limit_str)("l", std::locale("").name()));
       in.imbue(std::locale(""));
       double limit{0};
       in >> limit;
+      EOS_ASSERT(limit >= 0, plugin_config_exception, "block sync rate limit must be positive: ${limit}", ("limit", limit_str));
```

Comment:
I think this should be `limit >= 0.0`. Also for below, drop the `f` as that is `float`. `double` is undecorated.

Reply:
Updated.